### PR TITLE
contact-form

### PR DESCRIPTION
**Pull request by Yoz21**
# This is what we do in this project:

> we have worked on the contact form's HTML and CSS

> we checked for any lintern error

>we have used all the necessary font and styles from Figma

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,183 +1,376 @@
...
+      </div>
+    </section>
+    <section class="contact-form">
+      <div class="the-contact">
+        <p class="contact-p">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+
+        <form action="https://formspree.io/f/" method="post">
```
Comment (mateo951) [inline index.html]: Kindly make sure that your formspree form is working properly. 

Hint 🤫 Check your action URL (Is missing something)

Captura

File: index.html [hunk 2]
```diff
@@ -1,183 +1,376 @@
...
+              </label>
+            </p>
+            <label>
+              <textarea
+                name="comment"
+                required
+                minlength="2"
+                maxlength="500"
+                placeholder="Hello Vistor! Glad to see you!"
+              ></textarea>
```
Comment (mateo951) [inline index.html]: All your inputs have the propper type attribute 🔥

File: index.html [hunk 3]
```diff
@@ -1,183 +1,376 @@
...
+            <div class="name">
+              <label>
+                <input
+                  type="text"
+                  id="full-name"
+                  name="my-name"
+                  placeholder="Full Name"
+                  required
+                  minlength="2"
+                  maxlength="30"
```
Comment (mateo951) [inline index.html:343]: Nice work using the min-max length for the name entered value😎

File: index.html (hunk 2 above)
Comment (mateo951) [inline index.html]: Please try to follow the figma's design as much as possible 🤓 

#### Your design 
Captura2

#### Figma's 
Captura

What do you think if we use the same color on the text area to be as close as possible 🔥?

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi team, 

📝👌 **Your project highlights** 🔥😎 
- Linters are on point ✔
- Awesome PR 👌

Nice job so far! There are some changes to work on before you go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: index.html (hunk 2 above)
Comment (Yoz21) [reply index.html]: Thanks bro for this and all the comments.

File: index.html (hunk 1 above)
Comment (Whoistolu) [reply index.html]: - [ ] [REQUIRED] Please check your action URL . You have committed the form ID.
Your action value should be similar to `https://formspree.io/f/{form_id}`
Right now, your form is not still allowing submissions. 
form

Comment (Whoistolu) [inline index.html]: - [ ] [OPTIONAL] you have done a great job implementing some of the recommendations of the previous reviewer, however, you still need to stick to the design as much as possible.  You have omitted the last part of the form that has the list of social media icons. Also, fix the margin-top between the Get in touch `button` and the text area.
 
Your Implementation             |  Figma Design
:-------------------------:|:-------------------------:
your-design | figma2

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi team,
@Yoz21 and @daudi13 

You have done a great job so far, however, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the inline comments under the review for sections to improve._

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by rustam-tolipov — APPROVED** [review]
## Hi team👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**